### PR TITLE
[FEAT] reduce the subfru data side storage

### PR DESCRIPTION
**Pull request by sindhu-106**
Signed-off-by: sindhuja <sindhuja.gajula@intel.com>

### CODE REVIEW

**Review by anpatel1 — COMMENTED** [review]
You didnt change the column sizes back to what is needed. Shouldnt you be making that change?

File: inventory/src/test/groovy/com/intel/dai/inventory/RawInventoryDataIngesterSpec.groovy [hunk 1]
```diff
@@ -151,4 +158,13 @@ class RawInventoryDataIngesterSpec extends Specification {
         expect:
         RawInventoryDataIngester.constructDimm('bad_json') == null
     }
+
+    String json = "{\"hostname\": \"test-location\",\n" +
+            "          \"timestamp\": 1660247835145,\n" +
+            "          \"values\": {\n" +
+            "            \n" +
+            "           \"cpu1.id1\":\"value1\"\n" +
```
Comment (yhuan47) [inline inventory/src/test/groovy/com/intel/dai/inventory/RawInventoryDataIngesterSpec.groovy:166]: Could u add at least 2+ items for each CPU in test cases? cpu1.id1, cpu1.id2, cpu2.id1, cpu2.id2

**Review by yhuan47 — DISMISSED** [review]

File: inventory/src/test/groovy/com/intel/dai/inventory/RawInventoryDataIngesterSpec.groovy (hunk 1 above)
Comment (sindhu-106) [reply inventory/src/test/groovy/com/intel/dai/inventory/RawInventoryDataIngesterSpec.groovy:166]: sure Leo. I can add 1 more

**Comment by sindhu-106** [comment]
> You didnt change the column sizes back to what is needed. Shouldnt you be making that change?



> You didnt change the column sizes back to what is needed. Shouldnt you be making that change?

The code review is not merged as this is ready. I closed the PR.
PR reference: https://github.com/unifiedcontrolsystem/dai-ds/pull/722

**Review by anpatel1 — APPROVED** [review]

**Review by nbwhite — APPROVED** [review]

**Pull request merged**